app/assets/stylesheets/credits.scss

```diff
@@ -247,6 +247,15 @@
     border-radius: 3px;
     text-align: center;
 
+    @media screen and ( min-width:949px ) {
+      margin: 67px auto;
+      border-radius: 0;
+    }
+
+    @media screen and ( min-width:380px ) {
+      padding: 30px 20px;
```

Comment:
I think we can easily drop changing padding in media queries, and just make more room for the button overall:
On `.existing-credits` I'd probably decrease the padding to be:

```
padding: 24px 8px;
```

^ this would give us more room for button itself.

Additionally, just to be sure button won't break that terribly, I'd apply to it.
```
white-space: nowrap;
```